main.py

```diff
@@ -24,20 +24,71 @@
 
 WORKFLOWS: dict[str, list[StageDecorator]] = {
     'pre_alignment': [FastQCMultiQC],
-    'seqr_loader': [MtToEs, GvcfMultiQC, CramMultiQC],
+    'seqr_loader': [AnnotateDataset, MtToEs, GvcfMultiQC, CramMultiQC],
     'large_cohort': [LoadVqsr, Frequencies, GvcfMultiQC, CramMultiQC],
     'gatk_sv': [GatherBatchEvidence],
 }
 
 
-@click.command()
-@click.argument('workflow', type=click.Choice(list(WORKFLOWS.keys())))
-@click.option('--config', 'config_paths', multiple=True)
-def main(workflow: str, config_paths: list[str]):
+@click.command(no_args_is_help=True)
+@click.argument('workflow', required=False)
+@click.option(
+    '--config',
+    'config_paths',
+    multiple=True,
+    type=click.Path(exists=True),
+    help='Add configuration files to the files specified $CPG_CONFIG_PATH.'
+    'Configs are merged left to right, meaning the rightmost file has the'
+    'highest priority.',
+)
+@click.option(
+    '--list-workflows',
+    'list_workflows',
+    is_flag=True,
+    help='Only list possible values for WORKFLOW (and available last stages)',
+)
+@click.option(
+    '--list-last-stages',
+    'list_last_stages',
+    is_flag=True,
+    help='Only list possible end stages for a workflow, that can be specified '
+    'with `workflow/last_stages` in config',
```

Comment:
Is this correct that we can now only select from these as 'last_stages'? (I was not clear where the logic ended up in that previous PR).

Reply:
Yeah. The previous PR would have allowed to select any stages for `last_stages` (including `Align`, etc), but it didn't go in